conf/modules.config

```diff
@@ -150,6 +151,15 @@ process {
         ]
     }
 
+    withName: FEATURECOUNTS {
```

Comment:
I suppose this works, but shouldn't one use `.*:` syntax?
Another thing: The process should probably be called `FEATURECOUNTS_CDS` in preparation for possible other fc processes and to be more transparent.